docker/pipeline/Dockerfile.amd64

```diff
@@ -8,7 +8,7 @@
 # SPDX-License-Identifier: MPL-2.0
 ##############################################################################
 
-FROM node:12.13.0 as base
+FROM mhart/alpine-node:12.13.0 as base
```

Comment:
Had to change this because alpine-node and node have different libc implementations and Dapr depends on grpc which is a C binding compiled as a node module.